nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2226,10 +2226,10 @@ async fn instance_disk_detach(
 /// by creation date, with the most recent images appearing first.
```

Comment:
Does this comment still make sense? I'm wary of saying something like _List system images_ given @kc8apf's [comment](https://matrix.to/#/!YNYPOVxjAUeXksTcRj:oxide.computer/$aaeck8NO-QCjNELtMrJ7xhskvgKsOS42RbTsqBRn8Wk?via=oxide.computer&via=unix.house&via=timecube.club) on `system images` being a distinct thing that would be confusing to be conflated with `global images`. I could update it to say _List system-wide images_ which does a better job of capturing the intent.